src/core/server/bootstrap.ts

```diff
@@ -66,6 +66,12 @@ export async function bootstrap({
 
   const root = new Root(rawConfigService, env, onRootShutdown);
 
+  if (cliArgs.optimize) {
+    const cliLogger = root.logger.get('cli');
+    cliLogger.info('Optimization no longer required.');
```

Comment:
This log line doesn't appear to actually work.

Reply:
Yeah, seems the log just buffers output until the server is started which makes sense, but since we're not starting the server I'm just going to remove the logging and rely on the release notes and --help